tools/InterfaceGenerator/generator/generators/SmartFactoryBase.py

```diff
@@ -12,7 +12,17 @@
 import string
 import uuid
 
-from generator import Model
+from model.array import Array
```

Comment:
this code `tools/InterfaceGenerator/generator/generators/SmartFactoryBase.py`
called from `tools/InterfaceGenerator/Generator.py`, where we adding appropriate paths:
```
ROOT = Path(__file__).absolute().parents[1]
sys.path.append(ROOT.joinpath('rpc_spec/InterfaceParser').as_posix())
```
and after this imports wrapped into try/catch:
```
try:
    ...
except ModuleNotFoundError as error:
```
so if issue with submodule appear it will be cached in Generator.py, so there is no options that such issue appear in SmartFactoryBase.py

Reply:
Oh very good, thank you for the explanation. 

Would it be possible to update the error message with some extra information?

```
Please make sure the RPC Spec Generator Submodule is initialized.
Check the sdl_core source directory in `tools/rpc_spec`.
Try running:
git submodule init
git submodule update
```

Reply:
extra information added